lib/idl/check_inputs.pro

```diff
@@ -36,7 +36,8 @@ PRO check_inputs, inputs
               calc_dcx:zero_int, calc_halo:zero_int, calc_cold:zero_int, $
 	      calc_pnpa:zero_int, calc_pfida:zero_int, $
               calc_brems:zero_int, calc_birth:zero_int, calc_neutron:zero_int,$
-              calc_fida_wght:zero_int, calc_npa_wght:zero_int}
+              calc_fida_wght:zero_int, calc_npa_wght:zero_int, $
+              adaptive:zero_int, p_adaptive:zero_int, split_tol:zero_double, max_cell_splits:zero_int}
```

Comment:
Why have a separate switch for the passive grid?

Reply:
It was suggested that users be able to select separate parameters for the beam grid and passive grid e.g. dene for track and denn for track_cylindrical